File: python/DP1/1003.py
import sys
sys.setrecursionlimit(2500)

if __name__ == "__main__":
    t = int(input())
for i in range(t):
    n = int(input())
    zero = 1
    one = 0
    tmp = 0
    for _ in range(n):
        tmp = one
        one = one + zero
        zero = tmp
    print(zero, one)

File: Go/DP1/1003.go
package main

import "fmt"

func main() {
	//reader := bufio.NewReader(os.Stdin)
	var tc int = 0
	fmt.Scan(&tc)
	for i := 0; i < tc; i++ {
		n := 0
		fmt.Scan(&n)
		zero := 1
		one := 0
		tmp := 0
		for j := 0; j < n; j++ {
			tmp = one
			one = one + zero
			zero = tmp
		}
		fmt.Println(zero, one)
	}
}
